### DISCUSSION TITLE
Beginner Question: move Z2M installation without re-pairing devices?

### DISCUSSION THREAD

**Discussion by guenterbe** [Koenkk/zigbee2mqtt#19335] [Devices]
Hi *,

The title says it all. I want to move the zigbe2mqtt "installation/configuration" from my workstation to my notebook. Is that possible without having to repair all zigbee devices?

Is it sufficient to only transfer the data/configuration.yaml? I cannot see any pairing information in that file.

(I use the Sonoff ZigBee 3.0 USB dongle, which is accessed via a serial COM port - in case that matters. The same USB Dongle will also be used on the other computer)

Thanks in advance from a complete beginner!

**Comment by Koenkk** [comment]
Yes its possible, just make sure to copy the whole data directory (not just the configuration.yaml)
